tests/functional-tests/utils/scale_operator_object_function.py

```diff
@@ -0,0 +1,537 @@
+        api_version="v1",
+        kind="Secret",
+        metadata=secret_metadata,
+        data=secret_data
+    )
+    try:
+        LOGGER.info(f'creating secret {secret_name} with {str(secret_data)}')
+        secret_api_response = secret_api_instance.create_namespaced_secret(
+            namespace=namespace_value,
+            body=secret_body,
+            pretty=True
+        )
+        LOGGER.debug(str(secret_api_response))
+    except ApiException as e:
+        LOGGER.error(
+            f"Exception when calling CoreV1Api->create_namespaced_secret: {e}")
+        assert False
+
+
+def delete_secret(secret_name):
+    """
+    delete secret secret_name
+
+    Args:
+       param1: secret_name - name of secret to be deleted
+
+    Returns:
+       None
+
+    Raises:
+        Raises an exception on kubernetes client api failure and asserts
+
+    """
+
+    delete_secret_api_instance = client.CoreV1Api()
+    try:
+        delete_secret_api_response = delete_secret_api_instance.delete_namespaced_secret(
+            name=secret_name, namespace=namespace_value, pretty=True)
+        LOGGER.info(f'secret {secret_name} deleted')
+        LOGGER.debug(str(delete_secret_api_response))
+    except ApiException as e:
+        LOGGER.error(
+            f"Exception when calling CoreV1Api->delete_namespaced_secret: {e}")
+        assert False
+
+
+def check_secret_exists(secret_name):
+    """
+    Checks secret secret_name exists or not
+
+    Args:
+       param1: secret_name - name of secret to be checked
+
+    Returns:
+       return True  , if secret exists
+       return False , if secret does not exists
+
+    Raises:
+        None
+
+    """
+
+    api_instance = client.CoreV1Api()
+    try:
+        api_response = api_instance.read_namespaced_secret(
+            name=secret_name, namespace=namespace_value, pretty=True)
+        LOGGER.debug(str(api_response))
+        LOGGER.info(f'secret {secret_name} exists')
+        return True
+    except ApiException:
+        LOGGER.info(f'secret {secret_name} does not exist')
+        return False
+
+
+def check_secret_is_deleted(secret_name):
+    """
+    checks secret deleted or not
+    if secret not deleted in 120 seconds , asserts
+
+    Args:
+       param1: secret_name - name of secret to be checked
+    """
+    var = True
+    count = 12
+    api_instance = client.CoreV1Api()
+    while (var and count > 0):
+        try:
+            api_response = api_instance.read_namespaced_secret(
+                name=secret_name, namespace=namespace_value, pretty=True)
+            LOGGER.info("still deleting secret")
+            LOGGER.debug(str(api_response))
+            count = count-1
+            time.sleep(10)
+        except ApiException:
+            LOGGER.info("secret deleted")
+            var = False
+
+    if count <= 0:
+        LOGGER.error("secret is not deleted")
+        assert False
+
+
+def create_configmap(file_path, make_cacert_wrong):

```

Comment:
When is  this used? The operator shouldn't need the config map to be  created.

Reply:
This configmap is being created for cacert file for SSL testcases.